optimization/imagify-bulk-buffer-size/imagify-bulk-buffer-size.php

```diff
@@ -1,32 +1,36 @@
 <?php
 /**
- * Plugin Name: Imagify | Change Bulk Buffer Size
- * Description: Helps to avoid CPU issues during bulk optimization.
- * Plugin URI:  https://github.com/wp-media/imagify-helpers/tree/master/optimization/imagify-bulk-buffer-size/
- * Author:      Imagify Support Team
- * Author URI:  http://imagify.io/
- * License:     GNU General Public License v2 or later
- * License URI: http://www.gnu.org/licenses/gpl-2.0.html
+ * Plugin Name:  Imagify | Change Bulk Buffer Size
+ * Description:  Helps to avoid CPU issues during bulk optimization.
+ * Plugin URI:   https://github.com/wp-media/imagify-helpers/tree/master/optimization/imagify-bulk-buffer-size/
+ * Version:      1.1
+ * Requires PHP: 5.3
+ * Author:       Imagify Support Team
+ * Author URI:   http://imagify.io/
+ * License:      GPLv2
+ * License URI:  http://www.gnu.org/licenses/gpl-2.0.html
  *
- * Copyright SAS WP MEDIA 2018
+ * Copyright 2020 WP Media
+ *
+ * @package WP-Media\ImagifyPluginHelpers\BulkBufferSize
  */
-namespace ImagifyPlugin\Helpers\optimization\bulk;
 
-defined( 'ABSPATH' ) or die();
+namespace ImagifyPluginHelpers\Optimization\BulkBufferSize;
+
+defined( 'ABSPATH' ) || die();
 
 add_filter( 'imagify_bulk_buffer_sizes', __NAMESPACE__ . '\buffer_sizes', 10, 3 );
```

Comment:
The priority and number of arguments can be omitted. Why? The callback `buffer_sizes` only uses 1 argument.